src/main/kotlin/lotto/Main.kt

```diff
@@ -0,0 +1,25 @@
+package lotto
+
+import lotto.domain.IssuedLottos
+import lotto.domain.Lotto
+import lotto.domain.LottoMatchStat
+import lotto.domain.LottoSeller
+import lotto.view.InputView
+import lotto.view.ResultView
+
+fun main() {
+    val inputView = InputView()
+    val resultView = ResultView()
+    val lottoSeller = LottoSeller()
+
+    val seedMoney: Int = inputView.inputSeedMoney()
+    val issuedLottos: IssuedLottos = lottoSeller.sell(seedMoney)
+    resultView.outputIssuedLottos(issuedLottos)
+
+    val winningNumbers: Set<Int> = inputView.inputWinningNumbers()
+    val lottoMatchStat: LottoMatchStat = issuedLottos.check(Lotto(winningNumbers))
+    resultView.outputLottoMatchStat(lottoMatchStat)
+
+    val earningsRate: Double = lottoMatchStat.calculateEarningsRate(seedMoney)
+    resultView.outputEarningsRate(earningsRate)
```

Comment:
```suggestion
    val seedMoney = inputView.inputSeedMoney()
    val issuedLottos = lottoSeller.sell(seedMoney)
    resultView.outputIssuedLottos(issuedLottos)

    val winningNumbers = inputView.inputWinningNumbers()
    val lottoMatchStat = issuedLottos.check(Lotto(winningNumbers))
    resultView.outputLottoMatchStat(lottoMatchStat)

    val earningsRate = lottoMatchStat.calculateEarningsRate(seedMoney)
    resultView.outputEarningsRate(earningsRate)
```

개인적으로 코틀린의 타입 추론에 대해서 타입은 생략해도 좋다고 봅니다.

Reply:
개인적으로 클래스 선언 혹은 팩토리 메서드를 사용하는 경우가 아니라면 타입 파악을 위해 타입을 붙이는 편입니다.

Reply:
오호 그렇군요! 저 같은 경우에는 메서드 제외하고는 타입을 모두 생략하고 있어서요! 이 부분은 스타일 차이겠군요 😊